packages/excalidraw/change.ts

```diff
@@ -626,6 +629,18 @@ export class AppStateChange implements Change<AppState> {
             );
 
             break;
+          case "croppingElementId": {
+            const croppingElementId = nextAppState[key];
+            const element =
+              croppingElementId && nextElements.get(croppingElementId);
+
+            if (element && !element.isDeleted) {
+              visibleDifferenceFlag.value = true;
+            } else {
+              nextAppState[key] = null;
+            }
+            break;
+          }
```

Comment:
@Mrazator not sure about this, though the below case may be unrelated (albeit it doesn't happen in similar cases such as linearElementEditor):

1. select image (-> adds undo delta for selection)
2. enter crop editor (-> adds undo delta for crop editor)
3. undo
4. -> skips the selection delta (1) as well ... why?